### PR TITLE
ECCI-474: Added changes for the events listing view block

### PR DESCRIPTION
**Pull request by kurtisRogers-Nomensa**
### What have I done?
- made card look like the news card
- stacks on mobile / columns display on desktop
- added date field in and styled based on image in ticket

### CODE REVIEW

File: config/default/contact.form.personal.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+uuid: e4850269-5e38-4640-8b02-15bad93a45fa
```
Comment (MatthewWilkes) [inline config/default/contact.form.personal.yml:1]: @kurtisRogers-Nomensa I know you said you did an import and an export, but you've got lots of changes in here unrelated to events listings. I really want commits to be specific to the intended changes.

File: config/default/webform.webform.feedback_form.yml [hunk 2]
```diff
@@ -63,11 +63,9 @@ elements: |-
       '#description': |-
         Please do not enter any personal information in this space
         <div id="extension-host">&nbsp;</div>
-  have_you_liked_our_new_intranet_site_:
-    '#type': textfield
-    '#title': 'Have you liked our new Intranet site? '
-    '#description': 'Please give your feed back&nbsp;'
-    '#required': true
+    do_you_love_this_website_so_much_:
```
Comment (MatthewWilkes) [inline config/default/webform.webform.feedback_form.yml:66]: This in particular feels wrong. Is webform excluded from config import? These both look like non-production placeholders and shouldn't be changing in this PR.

File: config/default/webform.webform.testing_if_this_works.yml [hunk 3]
```diff
@@ -0,0 +1,200 @@
+uuid: f9aae8e2-63c5-42e0-87e1-65128f4981aa
```
Comment (MatthewWilkes) [inline config/default/webform.webform.testing_if_this_works.yml:1]: This definitely shouldn't be added.

File: themes/custom/essex/css/preliminary.css [hunk 4]
```diff
@@ -119,3 +119,39 @@
     padding: calc(var(--spacing) * 2.25) calc(var(--spacing) * 5);
   }
 }
+
+/* Added for  */
```
Comment (MatthewWilkes) [inline themes/custom/essex/css/preliminary.css:123]: This comment is incomplete

**Review by MatthewWilkes — CHANGES_REQUESTED** [review]

File: themes/custom/essex/templates/content/node--localgov-event--card.html.twig [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    <div class="card-content">
+      {{ title_prefix }}
+      {% if label and not page %}
+        <h3{{ title_attributes }}>
+          <a href="{{ url }}" rel="bookmark">{{ label }}</a>
+        </h3>
+      {% endif %}
+      {{ title_suffix }}
+
+      {{ content.localgov_event_date }}
```
Comment (Polynya) [inline themes/custom/essex/templates/content/node--localgov-event--card.html.twig:102]: Use the time element, similar to localgov_news:
```
    <time datetime="{{ node.localgov_news_date.value }}" class="news-article__metadata-item news-article__metadata-item--date">
      {{ content.localgov_news_date }}
    </time>
```
https://github.com/localgovdrupal/localgov_news/blob/2.x/templates/node--localgov-news-article--full.html.twig#L84

Comment (kurtisRogers-Nomensa) [reply themes/custom/essex/templates/content/node--localgov-event--card.html.twig]: 👍

**Comment by MatthewWilkes** [comment]
@kurtisRogers-Nomensa Just checking, the base branch here has changed away from develop and then back. We sure it should be going into dev?

**Comment by Polynya** [comment]
I'm pretty sure this ticket isn't for the release on 15th November. Let's check with Anastasia on Monday.

I'm going to remove the unwanted config from this PR.

**Comment by kurtisRogers-Nomensa** [comment]
> @kurtisRogers-Nomensa Just checking, the base branch here has changed away from develop and then back. We sure it should be going into dev?

Yeah mate Anastasia doesn't want this in the `release/202311` release

**Comment by kurtisRogers-Nomensa** [comment]
> I'm pretty sure this ticket isn't for the release on 15th November. Let's check with Anastasia on Monday.
> 
> I'm going to remove the unwanted config from this PR.

Thanks mate

**Comment by AnthonySGeorge** [comment]
There seems to be the same css selectors in both css files e.g. .featured-news__card article.
Can the two files be consolidated?

**Pull request closed without merging**